dbml-homepage/docs/js-module/connector.md

```diff
@@ -0,0 +1,56 @@
+---
+title: '@dbml/connector'
+---
+
+[![NPM](https://img.shields.io/npm/v/@dbml/connector)](https://www.npmjs.com/package/@dbml/connector)
+
+This package is responsible for connecting to a database on your local environment and fetch its schema information.
+
+## Supported Databases
+
+- PostgreSQL
+- MySQL
+- MSSQL
+
+## Prerequisites
+
+Before you begin, ensure you have met the following requirements:
+
+- **Node.js**: You need to have Node.js version 18.0.0 or higher installed.
+
+## Installation
+
+```bash npm2yarn
+npm install @dbml/connector
+```
+
+## APIs
+
+```javascript
+const { connector } = require('@dbml/connector');
+```
+
+### `fetchSchemaJson(connection, format)`
```

Comment:
I see that the `@dbml/core` has different writing styles for these named exports (e.g. `importer`).
![image](https://github.com/user-attachments/assets/31d4ab9a-d671-4906-b888-e8691831c14a)

Please move `fetchSchemaJson` to a sub-section of `connector` to ensure the writing styles are consistent.

Reply:
Fixed in commit https://github.com/holistics/dbml/pull/615/commits/790438dd7ef00ab26cbd219a7ffe50634ddc5923